lib/jenkins_api_client/cli/node.rb

```diff
@@ -58,6 +59,79 @@ def print_general_attributes
         puts table
       end
 
+      desc "create", "creates new node(s)"
+      # CLI command that creates new node
+      option :json
+      option :name
+      option :slave_host
+      option :credentials_id
+      option :private_key_file
+      option :executors
+      option :labels
+      def create
+        @client = Helper.setup(parent_options)
+        if options[:json]
+          json_file = "#{options[:json]}" 
+          file = File.read(json_file)
+          node_details = JSON.parse(file)
+          node_label = node_details['node_label']
+          executors = node_details['executors']
+          template_detail_hash = node_details['template_detail']
+          host_list_array = node_details['host_list']
+          host_list_array.each_with_index {|val, index|
+          host_name = "#{val}"
+          if template_detail_hash['guest_user_credentials_id'] != ""
+            credentials_id = template_detail_hash['guest_user_credentials_id']
+          else
+            puts "failed to find a credentials_id for "+template_detail_hash['guest_username']+"\nPlease add this user in jenkins and #{options[:json]}"
+        end
+        @client.node.create_dumb_slave(
+        :name => host_name,
+        :slave_host => host_name,
+        :credentials_id => credentials_id,
+        :private_key_file => "",
+        :executors => executors ? (executors) : (12),
+        :labels => node_label)
+    }
```

Comment:
fixed. also fixed line 93. checking if executors is empty and then assigning 2 executors instead of 12